.travis.yml

```diff
@@ -5,11 +5,18 @@ rust:
 - stable
 - nightly
 
+env:
+- 
```

Comment:
nit: extra line

Reply:
Without this it will only build on android. (Unless I've misuderstood)

Reply:
ah, right.